monkestation/code/_HELPERS/unsorted.dm

```diff
@@ -0,0 +1,13 @@
+//Used to get a random untouched locker, created for the Stowaway trait
+/proc/get_untouched_unlocked_closed_locker() //I've seen worse proc names
+	var/list/picked_lockers = list()
+	var/turf/object_location
+	for(var/obj/structure/closet/find_closet in world)				//Get crates
+		if(!istype(find_closet,/obj/structure/closet/secure_closet))//But not the locked crates
+			object_location = get_turf(find_closet) 				//Get the location it's at
+			if(object_location) 									//If it can't read a Z on the next step, it will error out. Needs a separate check.
+				if(is_station_level(object_location.z) && !find_closet.opened && !find_closet.GetComponent(/datum/component/forensics)) //Untouched, on the station and closed.
```

Comment:
honestly, I would allow it to be spawned in lockers that have been touched. I think it'll be funny to have the chance to close a locker, and then suddenly some drunk guy pops out of it.

Reply:
The reason I don't want any touched lockers is because there is a chance that someone will throw a locker into the SM chamber or tesla containment, something like that.
It's to catch the really nasty round ending edge cases like that.

Reply:
Well, someone can still teleport a locker and NOT get any fingerprints on it, I'm sure. This also doesn't account for "what if someone put a SM on top of the locker" and "Oh no this room is depressurized". Honestly, spawning in bad places is something that happens, and something that the person acknowledges when picking this trait.

Reply:
Not possible to drag or push a locker without fingerprinting since it's checking for the existing component rather than player-scannable fingerprints.

I understand it's meant to be a hardmode trait, but I don't like the idea of completely screwing over a player with an instant death because someone decided to move every locker on the station into a dangerous area.

Depressurized areas at least have a chance to escape and a SM directly on top of the locker will actually not kill. It dusts upon touching or running into it rather than just existing on the same tile.